dind/Dockerfile

```diff
@@ -1,17 +1,20 @@
 FROM alpine:3.5
 
-ENV DOCKER_VERSION=1.13.1 \
-  DOCKER_COMPOSE_VERSION=1.20.1 \
+ENV DOCKER_VERSION=18.09.9 \
```

Comment:
17 major version bumps??

Reply:
you how it is... they just jumped versions from 1.13 to 17.x for year based versions.  So it's really only 2 major versions newer.  Still it's 2018 which is still old and I'm thinking I might try out a newer version to see if it just works